pandas/util/_validators.py

```diff
@@ -205,9 +205,39 @@ def validate_args_and_kwargs(fname, args, kwargs, max_fname_arg_count, compat_ar
     validate_kwargs(fname, kwargs, compat_args)
 
 
-def validate_bool_kwarg(value, arg_name):
-    """ Ensures that argument passed in arg_name is of type bool. """
-    if not (is_bool(value) or value is None):
+def validate_bool_kwarg(value, arg_name, none_allowed=True, int_allowed=False):
```

Comment:
remove the int_allowed (its not)

Reply:
This is required for ``validate_ascending`` as int is actually considered as a valid boolean there.
Previously I had this check in ``validate_ascending`` itself, but I was requested to use ``validate_boolean``, so I extended it in such a way.

It may be necessary to maybe restrict ``ascending`` to use of booleans only (no integers), but this is a breaking change and should probably be discussed first.